goal_src/engine/camera/cam-states.gc

```diff
@@ -2324,11 +2326,16 @@
       ((< 160 arg0)
        (set! f0-0 (fmin arg1 (* 0.0125 (+ -160.0 (the float arg0)) arg1)))
        )
-      )
-    f0-0
+    )
+    ;; only used for Third-Person Vertical camera
+    (*
+      (if (and PC_PORT (-> *pc-settings* third-camera-vflip?)) 1.0 -1.0)
+      f0-0
     )
   )
+)
 
+;; main third-person camera
 (defbehavior cam-string-joystick camera-slave ()
   (set! (-> self options) (logand -257 (-> self options)))
   (let ((f28-0 (cam-dist-analog-input (the-as int (-> *cpad-list* cpads 0 righty)) (the-as float 0.05)))
```

Comment:
`cam-dist-analog-input` is the equivalent of `v3-analog-input` for `cam-string-joystick`, since it has a unique vertical camera control